pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -380,12 +386,12 @@ Given
 
 @verbatim{  f(x) = x + 10}
 
-then an algebra student simplifies @tt{f(7)} as follows:
+an algebra student simplifies @tt{f(7)} as follows:
 
 @verbatim{  f(7) = 7 + 10 = 17}
 
 The key step in this simplification is take the body of the defined
```

Comment:
Change 'is take" to "is to take" or "is: take"
?

Reply:
Yes, thanks!